qml/MessagePopup.qml

```diff
@@ -184,10 +248,10 @@ Popup {
                             msg = "@" + message.author + " " + msg
                         }
 
-                        uiBridge.postButton(msgid, msg)
+                        uiBridge.postButton(msgid, msg, visibility)
                         messageArea.clear()
                     }
-                }
+                }                
```

Comment:
Minor nitpick: try to avoid superfluous spaces at line ends. Most editors have an option to automatically do that for you. By no means a blocker though :smiley: